app.rb

```diff
@@ -1,133 +1,110 @@
-    option = gets.chomp
-
-    case option
-    when '1'
-      create_student
-    when '2'
-      create_teacher
-    else
-      puts 'Invalid input. Try again'
+  def create_person()
+    puts 'Do you want create student🧑‍🎓 (1) or teacher🧑‍🏫 (2)? [Enter the number]: '
+    person_type = gets.chomp.to_i
+    case person_type
+    when 1
+      print 'Enter student name: '
+      name = gets.chomp
+      print 'Enter age: '
+      age = gets.chomp
+      @people.push(Student.new('classroom', age, name))
+      puts "#{name.capitalize} added successfully 💓"
+    when 2
+      print 'Enter teacher name: '
+      name = gets.chomp
+      print 'Enter age: '
+      age = gets.chomp
+      print 'Enter specialization: '
+      specialization = gets.chomp
+      @people.push(Teacher.new(specialization, age, name))
+      puts "#{name.capitalize} added successfully 💓"
     end
   end
 
-  def create_student
-    puts 'Create a new student'
-    print 'Enter student age: '
-    age = gets.chomp.to_i
-    print 'Enter name: '
-    name = gets.chomp
-    print 'Has parent permission? [Y/N]: '
-    parent_permission = gets.chomp.downcase
-    case parent_permission
-    when 'n'
-      student = Student.new('undefined', age, name, parent_permission: false)
-      @persons << student
-      puts 'Student doesnt have parent permission, cant rent books'
-    when 'y'
-      student = Student.new('undefined', age, name, parent_permission: true)
-      @persons << student
-      puts 'Student created successfully'
-    end
-  end
-
-  def create_teacher
-    puts 'Create a new teacher'
-    print 'Enter teacher age: '
-    age = gets.chomp.to_i
-    print 'Enter teacher name: '
-    name = gets.chomp
-    print 'Enter teacher specialization: '
-    specialization = gets.chomp
-    teacher = Teacher.new(specialization, age, name)
-    @persons.push(teacher)
-    puts 'Teacher created successfully'
-  end
-
-  def create_book()
-    puts 'Create a new book'
-    print 'Enter title: '
-    title = gets.chomp
-    print 'Enter author: '
-    author = gets
-    book = Book.new(title, author)
-    @books.push(book)
-    puts "Book #{title} created successfully."
-  end
-
-  def create_rental
-    puts 'Select which book you want to rent by entering its number'
-    @books.each_with_index { |book, index| puts "#{index}) Title: #{book.title}, Author: #{book.author}" }
-
-    book_id = gets.chomp.to_i
-
-    puts 'Select a person from the list by its number'
-    @persons.each_with_index do |person, index|
-      puts "#{index}) [#{person.class.name}] Name: #{person.name}, ID: #{person.id}, Age: #{person.age}"
-    end
-
-    person_id = gets.chomp.to_i
-
+  def create_rental()
+    puts 'Select a book from the following list by the number:📖'
+    list_all_books
+    book_number = gets.chomp.to_i
+    puts ' '
+    puts 'Select a person from the following list by the number:🧑‍🎓🧑‍🏫'
+    list_all_people
```

Comment:
### **Optional**
- [ ] Please try as much as possible to use `SOLID`  principles to refactor their classes. [SEE THIS ARTICLE](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/lessons/solid.md#best-practices-and-solid)